.travis.yml

```
@@ -1,22 +1,19 @@
 language: python
 python:
-  - "3.5"
   - "3.6"
   - "3.7"
-  - "3.8"
+  - "3.9"
```

```suggestion
  - "3.8"
  - "3.9"
```
… or am I missing something?

---

Yeah, for the Python versions listed here, only the unittests are executed. But for Python 3.8 the coverage reports is generated, because Python3.8 is included explicitly below.

If we added `3.8` to this list, Travis would run the unittests for Python 3.8 again (like for the other versions).

---

I see.

I'm not familiar with Travis, so thanks for the explanation.

This is fine, then.